app/views/auth/index.html.erb

```diff
@@ -1,3 +1,3 @@
 <h1>Connect your Spotify account</h1>
 
-<a href="https://accounts.spotify.com/authorize?client_id=<%= @client_id %>&response_type=code&redirect_uri=<%= ERB::Util.url_encode(@redirect_uri) %>&scope=user-follow-modify,user-library-read,playlist-read-collaborative,playlist-modify-private,user-modify-playback-state,streaming,app-remote-control,user-read-private,user-read-birthdate,user-read-playback-state,playlist-read-private,user-top-read,user-read-email,playlist-modify-public,user-library-modify,user-follow-read,user-read-currently-playing,user-read-recently-played">Connect</a>
\ No newline at end of file
+<a href="https://accounts.spotify.com/authorize?client_id=<%= @client_id %>&response_type=code&redirect_uri=<%= ERB::Util.url_encode(@redirect_uri) %>&scope=user-follow-modify,user-library-read,playlist-read-collaborative,playlist-modify-private,user-modify-playback-state,streaming,app-remote-control,user-read-private,user-read-playback-state,playlist-read-private,user-top-read,user-read-email,playlist-modify-public,user-library-modify,user-follow-read,user-read-currently-playing,user-read-recently-played">Connect</a>
```

Comment:
Need to extract the scopes from this crazy-long string and put in an environment variable.

Reply:
Let's do this later - preferably NOT while livestreaming so as not to expose secret credentials. 🤫